poyoweb-server/src/index.ts

```diff
@@ -1,13 +1,15 @@
 import { Elysia } from "elysia";
 import { cors } from '@elysiajs/cors';
+import { ip } from "elysia-ip";
 
 import AuthRouter from "./auth/auth";
 
 const app = new Elysia()
   .use(cors({
-    origin: 'http://localhost:5173', // Allow SvelteKit frontend
+    origin: ['http://localhost:5173', 'http://localhost'], // Allow SvelteKit frontend
     methods: ['GET', 'POST', 'PUT', 'UPDATE', 'DELETE'], // Allowed methods
   }))
+  .use(ip())
```

Comment:
the ip middleware is added twice, once on the auth router, once here. maybe only do it once?